go-workout/edgedb.toml

```diff
@@ -0,0 +1,2 @@
+[edgedb]
+server-version = "2.0"
```

Comment:
does it work to have the `edgedb.toml` here and `dbschema` directories in sub directories?

Reply:
I don't believe so